src/Records/src/Main.java

```diff
@@ -0,0 +1,11 @@
+public class Main {
+    public static void main(String[] args) {
+        ex6();
+    }
+    public static void ex6() {
+        Student s1 = new Student("Yahli", 3);
+        System.out.println(s1.toString());
+        if (s1.isHighSchool(s1.Class())== true) System.out.println("The student is highschool");
```

Comment:
```suggestion
        if (s1.isHighSchool(s1.Class())) System.out.println("The student is highschool");
```
you don't need check if the boolean is true, the if already does it